src/components/dropdown/dropdown.js

```diff
@@ -0,0 +1,36 @@
+const defaultOption = { value: "", text: "선택해 주세요" };
+
+export default function createDropdownBox({
+  labelText,
+  id,
+  dropdownList,
+  required,
+}) {
+  const dropdownBox = createElement("div", {
+    className: ["form-item", `${required && "form-item--required"}`],
```

Comment:
사소하지만 지금은 `'false'`라는 클래스가 들어가겠네요! 
createElement 에서 [falsy](https://developer.mozilla.org/ko/docs/Glossary/Falsy)한 값을 필터링 하는 것 어떤가요?

```suggestion
    className: ["form-item", required && "form-item--required"],
```

Reply:
해당 부분 수정 했습니다. 일관성을 위해서, required에 기본값을 false로 주는 것도 집어 넣었어요.